crates/hir/src/lib.rs

```diff
@@ -6343,3 +6483,42 @@ where
         self(item)
     }
 }
+
+pub fn resolve_absolute_path(
```

Comment:
This method does not make sense, a crate name only exists as a name of a dependency edge. That is we can only resolve paths from within the context of some crate.